dymension/assetlist.json

```diff
@@ -33,6 +33,23 @@
         "website": "https://portal.dymension.xyz",
         "twitter": "https://twitter.com/dymension"
       }
+    },
+    {
+      "description": "The token from DRBE chain",
+      "denom_units": [
+        {
+          "denom": "ibc/5620289B0E1106C8A2421F212FEC4EB19E3CBA964662DB61754CCDE8FAAC29FF",
+          "exponent": 0
+        },
+        {
+          "denom": "DRBE",
+          "exponent": 6
+        }
+      ],
+      "base": "ibc/5620289B0E1106C8A2421F212FEC4EB19E3CBA964662DB61754CCDE8FAAC29FF",
```

Comment:
For ibc tokens, please specify type_asset ics20, and define the traces so we know where it comes from (this requires the ibc connection and counterparty chain to be registered)